cmd/tealdbg/cdtState.go

```diff
@@ -441,6 +442,7 @@ func makePreview(fields []fieldDesc) (prop []cdt.RuntimePropertyPreview) {
 }
 
 func makeIntPreview(n int) (prop []cdt.RuntimePropertyPreview) {
+	prop = make([]cdt.RuntimePropertyPreview, 0, n)
```

Comment:
I like that you preallocated the array.. unfortunately, if n is negative, the function would now panic.